mozjs-sys/src/jsimpls.rs

```diff
@@ -119,6 +112,13 @@ impl<T> JS::MutableHandle<T> {
     {
         unsafe { *self.ptr = v }
     }
+
+    /// The returned pointer is aliased by a pointer that the GC will read
```

Comment:
I've added a task to the tracking issue to update this comment if we don't convert the GC to only write via interior mutability.